src/test/kotlin/org/rust/cargo/RsWithToolchainTestBase.kt

```diff
@@ -77,6 +78,9 @@ abstract class RsWithToolchainTestBase : CodeInsightFixtureTestCase<ModuleFixtur
         if (disableMissedCacheAssertions) {
             RecursionManager.disableMissedCacheAssertions(testRootDisposable)
         }
+        findAnnotationInstance<UseOldResolve>()?.let {
+            DefMapService.setNewResolveEnabled(project, testRootDisposable, false)
+        }
```

Comment:
Why only `UseOldResolve`?
I'd extract code from `RsTestBase` for both annotations to util function and use it here as well